src/EditorFeatures/TestUtilities/TestObscuringTipManager.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System.ComponentModel.Composition;
+using Microsoft.VisualStudio.Text.Editor;
+
+namespace Microsoft.CodeAnalysis.Editor.UnitTests
+{
+    // In 15.6 the editor (QuickInfo in particular) took a dependency on 
+    // IObscuringTipManager, which is only exported in VS editor layer.
+    // This is tracked by the editor bug #544569.
```

Comment:
Full URL to bug please. Is this being fixed in a newer version of the packages?

Reply:
Bug is currently marked to be fixed in 15.8: https://devdiv.visualstudio.com/DevDiv/_workitems?id=544569